hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -187,6 +207,14 @@ def make_figure_dirs(subfolder: str, parent_dir: Path) -> Path:
     return figures_dir
 
 
+def get_list_from_results_dict(results: ResultsType, results_key: ResultsKey) -> List[Any]:
+    """ Get a specific results list from the slide_level results dictionary
```

Comment:
can you please mention that we extract items from tensors  here so that it's compatible with inputs formats of scikit learn functions?

Reply:
done, thanks